src/components/walkthroughDetails/walkthroughDetails.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card, CardBody, TextContent } from '@patternfly/react-core';
+import { connect, reduxActions } from '../../redux';
+
+class WalkthroughDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    this.props.getWalkthroughInfo();
+  }
+
+  render() {
+    const { walkthroughInfo } = this.props;
+    return (
+      <Card>
+        <CardBody>
+          <TextContent className="integr8ly-walkthrough-resources pf-u-pl-md">
+            <h2>About this walkthrough</h2>
+            <h3>Details</h3>
+            <div className="pf-u-pb-sm">
+              <div className="pf-u-display-flex pf-u-justify-content-space-between">
+                <div>Source: </div>
+                <div>
+                  <a href={walkthroughInfo.gitUrl} target="_blank" rel="noopener noreferrer">
+                    {walkthroughInfo.gitUrl === 'https://github.com/integr8ly/tutorial-web-app-walkthroughs.git'
```

Comment:
We could include this check/info from the `/about` endpoint if that'd be cleaner?